packages/flutter_test/lib/src/controller.dart

```diff
@@ -95,6 +104,104 @@ class SemanticsController {
     return result;
   }
 
+  /// Attemps to find a [SemanticsNode] with the given `action` as close to the
+  /// [SemanticsNode] of `finder` as possible.
+  ///
+  /// The [SemanticsNode] directly related to an [Element] is not necessarily
+  /// the one that can be acted upon. This method finds the directly related
+  /// node and checks whether it supports the given `action`. If the directly
+  /// related node does not support the given `action`, then the closest
+  /// descendent of that node that does support the `action` is returned.
+  ///
+  /// Will throw a [StateError] if:
+  /// * No results are returned from `finder`
+  /// * More than result is returned from `finder`
```

Comment:
nit: missing word after than?

Reply:
Good catch, I guess my brain just added that word in without letting me know it was missing. 😛